File: UaicLibraryWebSpa/src/app/models/profile/user-info.ts
export class UserInfo {
    public id: number;
    public fullName:string;
    public role: string;
    public readedBooks: number;
    public publishedBooks: number;
    public opennedBooks: number;
    public about: string
    constructor(id:number, fullName:string, role: string, readedBooks: number, publishedBooks: number, opennedBooks: number, about: string) {
        this.id = id;
        this.fullName = fullName;
        this.role = role;
        this.readedBooks = readedBooks;
        this.publishedBooks = publishedBooks;
        this.opennedBooks = opennedBooks;
        this.about = about;
     }
}

File: UaicLibrary.Domain/UserManagement/UserInfo.cs
namespace UaicLibrary.Domain.UserManagement
{
    public class UserInfo
    {
        public int Id { get; set; }
        public string FullName { get; set; }
        public string Role { get; set; }
        public int ReadedBooks { get; set; }
        public int PublishedBooks { get; set; }
        public int OpennedBooks { get; set; }
        public string About { get; set; }
    }
}
